### ISSUE TITLE
Input execd - cannot use command parameters (on Windows)

### ISSUE THREAD

**Issue by Trovalo** [influxdata/telegraf#7859]
### Relevant telegraf.conf:

```toml
# Global tags can be specified here in key="value" format.
[global_tags]

# Configuration for telegraf agent
[agent]
  interval = "10s"
  round_interval = true
  metric_batch_size = 1000
  metric_buffer_limit = 10000
  collection_jitter = "0s"
  flush_interval = "10s"
  flush_jitter = "0s"

  precision = ""

  debug = true
  # quiet = false

  logtarget = 'stderr'
  # logfile = ''
  # logfile_rotation_interval = "14d"
  # logfile_rotation_max_size = "0MB"
  # logfile_rotation_max_archives = 5

  hostname = ""
  omit_hostname = false

###################################
#         OUTPUT PLUGINS           
###################################

[[outputs.file]]
  ## Files to write to, "stdout" is a specially handled file.
  files = ['C:\Users\gluisotto\Desktop\test_Telegraf\execd_output.out']

  data_format = "influx"

###################################
#         INPUT PLUGINS           
###################################
[[inputs.execd]]

  ## use bat file
  #command = ['C:\Users\gluisotto\Desktop\test_Telegraf\StartPrintrandom.bat']
  ## use command without params (default values)
  command = ['Powershell.exe -File "C:\Users\gluisotto\Desktop\test_Telegraf\PrintRandom.ps1"']
  ## use command with param
  #command = ["Powershell.exe -File 'C:\Users\gluisotto\Desktop\test_Telegraf\PrintRandom.ps1' -MaxWaitSec 2 -LoopNum 4"]

  signal = "none"

  restart_delay = "10s"

  data_format = "influx"

```

### System info:
Telegraf 1.15 (nightly)

### Steps to reproduce:
Here you can find all the files I've used:
[test_Telegraf.zip](https://github.com/influxdata/telegraf/files/4948259/test_Telegraf.zip)

1. Execute the PowerShell script on its own, it works (with or without params)
2. Execute the PowerShell script using _execd_ and with parameters - it fails
3. Execute the PowerShell script using _execd_ and without parameters - it fails
4. Execute the PowerShell script using _execd_ and by using a .bat file - it works

the same syntax in the _exec_ plugin works fine
> command = ['Powershell.exe -File "C:\Users\gluisotto\Desktop\test_Telegraf\PrintRandom.ps1"']

in the execd input I get the following error:
```
2020-07-20T14:11:37Z I! Starting process: [Powershell.exe -File "C:\Users\gluisotto\Desktop\test_Telegraf\PrintRandom.ps1"]
2020-07-20T14:11:37Z E! [telegraf] Error running agent: starting input inputs.execd: Error starting process: exec: "Powershell.exe -File \"C:\\Users\\gluisotto\\Desktop\\test_Telegraf\\PrintRandom.ps1\"": file does not exist
```
Which to me is caused by some weird escapes inside the strings as you can see from the error

['Powershell.exe -File "C:\Users\gluisotto\Desktop\test_Telegraf\PrintRandom.ps1"']
### Expected behavior:
A PowerShell file can be invoked with parameters, it requires to use double quotes inside single quotes
### Actual behavior:
Error:
> Error running agent: starting input inputs.execd: Error starting process: exec: "Powershell.exe -File \"C:\\Users\\gluisotto\\Desktop\\test_Telegraf\\PrintRandom.ps1\"": file does not exist

### Additional info:

Error logs

**ps1 file without parameters**
```
{...}
2020-07-20T14:11:37Z D! [agent] Successfully connected to outputs.file
2020-07-20T14:11:37Z D! [agent] Starting service inputs
2020-07-20T14:11:37Z I! Starting process: [Powershell.exe -File "C:\Users\gluisotto\Desktop\test_Telegraf\PrintRandom.ps1"]
2020-07-20T14:11:37Z E! [telegraf] Error running agent: starting input inputs.execd: Error starting process: exec: "Powershell.exe -File \"C:\\Users\\gluisotto\\Desktop\\test_Telegraf\\PrintRandom.ps1\"": file does not exist
```

**ps1 file with parameters** (for this I should probably find a different approach though)
```
2020-07-20T14:18:19Z I! Starting Telegraf 
2020-07-20T14:18:19Z E! [telegraf] Error running agent: Error loading config file .\execd_TestConfig.conf: Error parsing data: line 50: invalid TOML syntax
```
a toml validator returns me this error:
> Expected EightHexadecimalDigits but "s" found.

**Comment by ssoroka** [comment]
Looks like the syntax of the command is incorrect in your config.

should be 
`command = ["Powershell.exe", "-File", "C:\Users\gluisotto\Desktop\test_Telegraf\PrintRandom.ps1"]`.

this seems to be a common enough error that we should check for this.

**Comment by Trovalo** [comment]
I was not even aware this syntax could work like that, so far I've used it only to execute more than one command in the _exec_ plugin.
as an example, I've used it to run the same script with 3 different input params:
```
command = [
  'Powershell.exe -File "C:\\temp\custom_script.ps1" "Srv1"',
  'Powershell.exe -File "C:\\temp\custom_script.ps1" "Srv2"'
  'Powershell.exe -File "C:\\temp\custom_script.ps1" "Srv3"'
]
```
Maybe I will make a PR to edit the exec and execd readme.md in order to point out how the command array works.

Can the _execd_ plugin run/manage more than one executable in the same "instance"? or is it better to have an _execd_ section for each executable?
